meta-resin-common/classes/image-resin.bbclass

```diff
@@ -193,6 +193,10 @@ resin_boot_dirgen_and_deploy () {
 
     echo "Install resin-boot in the rootfs..."
     cp -rvf ${RESIN_BOOT_WORKDIR} ${IMAGE_ROOTFS}/${RESIN_BOOT_FS_LABEL}
+
+    echo "Generating ${RESIN_BOOTFILES_LIST} ..."
+    mkdir -p $(dirname ${RESIN_BOOTFILES_LIST})
+    find ${RESIN_BOOT_WORKDIR} -type f | sed "s#${RESIN_BOOT_WORKDIR}##g" > ${RESIN_BOOTFILES_LIST}
```

Comment:
does it make sense to declare that default in image_types_resin.bbclass since we overwrite it here with the actual list of files present in the bootdir?

Reply:
I'm not sure I understand. image_types_resin doesn't know the list of files. It uses everything this function deploys in the boot partition work directory.

Reply:
I was saying why declare that default since it gets overwritten anyway.
Anyhow, lgtm'ed this

Reply:
What is "that"? This is a `find` command.

Reply:
I was saying since you overwrite it here in "this" file, I did not understand why you declare with those values "there": https://github.com/resin-os/meta-resin/pull/835/commits/474b9919d68c0e8ca1b4a97cc34fbf576ce1fc03

Where this "there" is the "that" above